scripts/setup.sh

```diff
@@ -28,13 +28,35 @@ OKTA_REGISTRY=${ARTIFACTORY_URL}/api/npm/npm-okta-master
 #  - https://github.com/yarnpkg/yarn/issues/5892
 #  - https://github.com/yarnpkg/yarn/issues/3330
 
-# Replace yarn artifactory with Okta's
+# Replace yarn registry with Okta's
+echo "Replacing $YARN_REGISTRY with $OKTA_REGISTRY within yarn.lock files..."
 sed -i "s#${YARN_REGISTRY}#${OKTA_REGISTRY}#" yarn.lock
+sed -i "s#${YARN_REGISTRY}#${OKTA_REGISTRY}#" packages/okta-angular/test/e2e/harness/yarn.lock
```

Comment:
Do we need to update the lock files in test directory too?

Reply:
yes